week4/assignment/src/constants/signupStep.ts

```diff
@@ -0,0 +1,7 @@
+export const SIGN_UP_STEP = {
+  CREATE_ID: 1,
+  CREATE_PASSWD: 2,
+  CREATE_NICKNAME: 3,
+} as const;
+
+export type SignUpStep = (typeof SIGN_UP_STEP)[keyof typeof SIGN_UP_STEP];
```

Comment:
```suggestion
export const enum SignUpStep {
  CREATE_ID = 1,
  CREATE_PASSWD = 2,
  CREATE_NICKNAME = 3,
}
```

enum을 사용하면 타입이 자동 생성됩니다.

https://github.com/toss/frontend-fundamentals/discussions/6#:~:text=export%20const%20~%20%3D%20%7B%20~~%20%7D%20as%20const%20%EC%99%80%20%EA%B0%99%EC%9D%80%20%EB%AC%B8%EB%B2%95%EC%9D%80%20%EC%95%84%EB%9E%98%EC%99%80%20%EA%B0%99%EC%9D%B4%20%EB%8C%80%EC%B2%B4%EB%90%A0%20%EC%88%98%20%EC%9E%88%EC%8A%B5%EB%8B%88%EB%8B%A4.

enum 사용에 대한 장단점도 있으니 한 번 읽으보시는거 추천드릴게요